counterfab/src/main/java/com/andremion/counterfab/CounterFab.java

```diff
@@ -134,6 +134,41 @@ public CounterFab(Context context, AttributeSet attrs, int defStyleAttr) {
         onCountChanged();
     }
 
+    public void setBadgeTypeface(Typeface typeface) {
```

Comment:
If we need some customization in the widget style, we need allow setting on Java and XML.
Beside that, for each changing that affects the drawing properties after the widget has been drawn, we need to call the invalidate method.